Tests/PerformanceTestingTests/SetTests.swift

```diff
@@ -9,136 +9,133 @@ import XCTest
 import PerformanceTesting
 
 class SetTests: PerformanceTestCase {
-/*
+
     // MARK: Helper functions.
 
     // Constructs a set of size `n` with linearly increasing elements.
-    let constructSizeNSet: Setup<Set<Int>> = { set, n in
-        set.reserveCapacity(Int(n))
-        for i in 0..<Int(n) {
+    func constructSet(size n: Int) -> Set<Int> {
+        var set = Set<Int>()
+        set.reserveCapacity(n)
+        for i in 0..<n {
             set.insert(i)
         }
+        return set;
+    }
+
+    // Random number in [0, upperLimit)
+    func randomInteger(_ upperLimit: Int = 1000) -> Int {
```

Comment:
We should make this an extension on `Int`, perhaps dovetailed with the other one. Something like this:

```Swift
extension Int {

    public static func random(min: Int = 0, max: Int = 1) -> Int {
        let (min, max) = ordered(min, max)
        let range = max - min
        return Int(arc4random_uniform(UInt32(range))) + min
    }
}
```